.travis.yml

```diff
@@ -7,8 +6,16 @@ python:
 
 jobs:
   include:
-    # OSX build removed due to long build startup delays
-    # Restore matrix job entry from d2d67fab to test OSX
+    - os: osx
+      osx_image: xcode6.4
+      language: bash
+      python: 3.6
+      env:
+        - DYLD_LIBRARY_PATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/
```

Comment:
I think this is not needed any more.  It was for llvm support, which has been moved to bears only.